troubleshooting/ETW/Readme.md

```diff
@@ -127,43 +127,69 @@ This file will not exceed the configured max size and will be circularly overwri
 
 #### Configuration
 
-Configuration is controlled by a file named `ApplicationInsightsDiagnostics.json`.
+Configuration is controlled by a file named `ApplicationInsightsDiagnostics.json` or environment variables.
```

Comment:
Specify the list of environment variables.

Reply:
Are we requiring that users should set all three environment variables?
I wonder if it would be easier for users if there was only one environment variable consisting of key/value pairs?
@rajkumar-rangaraj is there any recommendation/standard that we should follow?

Reply:
I agree. Tt is easier to manage if it is one environment variable with key/value pairs.

Reply:
key/value pairs sounds better!!
I am not sure what should be a proper name for holding the pairs.
"SelfDiagnosticsConfig"?

Reply:
The name of the config file is "ApplicationInsightsDiagnostics".
So to keep it similar, I would vote for "APPLICATIONINSIGHTS_DIAGNOSTICS".